Source/Modules/php.cxx

```diff
@@ -441,6 +475,15 @@ class PHP : public Language {
     Printf(s_header, "}\n");
     Printf(s_header, "#endif\n\n");
 
+    Printf(s_header,"#ifdef __GNUG__\n#define SWIG_remove(zv) delete zv\n");
+    Printf(s_header,"#else\n#define SWIG_remove(zv) free(zv)\n#endif\n\n");
+
+    Printf(s_header, "#define CALL_METHOD(name, retval, thisptr)                  \
+                      call_user_function(EG(function_table),thisptr,&name,retval,0,NULL TSRMLS_CC);\n\n");
+
+    Printf(s_header, "#define CALL_METHOD_PARAM_1(name, retval, thisptr, param)                  \
+                      call_user_function(EG(function_table),thisptr,&name,retval,1,&param TSRMLS_CC);\n\n");
```

Comment:
`TSRMLS_CC` isn't needed with PHP7.  None of the `TSRM` stuff is.  They're still present, but defined to "no op" values so that existing code using them still builds.  No point using them in new code.